repair/repair.cc

```diff
@@ -1409,6 +1431,10 @@ static future<> do_repair_ranges(lw_shared_ptr<repair_info> ri) {
                         return make_ready_future<stop_iteration>(stop_iteration::yes);
                     }
                 });
+            }).then([ri] {
+                if (ri->reason == streaming::stream_reason::bootstrap) {
+                    _node_ops_metrics.bootstrap_finished_ranges++;
+                }
```

Comment:
Maybe you can do `++_node_ops_metrics[reason].finished_ranges` to monitor all types.

Reply:
Normal repair is more complicated than the node operations. User can fire more repair jobs in parallel. I want to limit those to node operations for now. If we end up we can simplify with the  _node_ops_metrics[reason], we can easily factor this out. OK?

Reply:
Ok, but I do want to make sure the metrics are similar and work for both repair and other node operations.

Reply:
I agree that it is better to make progress metrics for repair and the rest of the operations look similar.